packages/nimble-components/src/wafer-map/specs/features/grid-dimension.md

```diff
@@ -0,0 +1,28 @@
+# Nimble Wafer Map Grid Dimensions
+
+## Problem Statement
+
+The Wafer Map displays dies on a canvas inside a circular border based on an internal grid which is calculated based on the data input.
+This approach is limiting the performance of the wafer and the grid dimensions can be erroneous if the displayed data is just a subset of a larger data set.
+To mitigate these problems, we will reveal three new API inputs which will allow the users to set the grid dimensions themselves, thus saving processing time and correcting inconsistencies generated by the lack of data context.
+
```

Comment:
Is this API effectively a way to programmatically zoom into a certain area of the wafer? Can you clarify what the interactive impact this has?

Can the user zoom out interactively and see more of the data? Can the zoom in interactively to a region of this subset of data?

Reply:
yes, essentially, it's a way to set the predefined zoom level of the data if you wish to see it that way. Right now, we are calculating the bounding box of the input data with the assumption that it is circular, and the missing data points are accidental. When the user is voluntary trying to display an incomplete subset for various purposes the data display gets warped. this change is a way to fix that.

the way you describe using it to zoom in and out programmatically is mostly a side effect. Do you recommend to discourage that usage by limiting the user input to being greater than the existing bounding box? or should we encourage it by filtering out the values outside the ones specified? Is there a different solution to our problem?